components/sendgrid/sources/common/http-based.js

```diff
@@ -0,0 +1,75 @@
+const base = require("./base");
+
+module.exports = {
+  ...base,
+  props: {
+    ...base.props,
+    http: "$.interface.http",
+  },
+  hooks: {
+    ...base.hooks,
+    async activate() {
```

Comment:
I think deactive/activate get called anytime props are updated for example... can we be a bit more forgiving here if we are the owners of the webhook?

Reply:
That's a good idea. We can check if the URL's match for example. Does that make sense?